tasks/main.yml

```diff
@@ -74,9 +73,29 @@
     group: root
     mode: "0755"
 
-
 - name: "Start and enable the Loki daemon"
   ansible.builtin.systemd:
     name: loki
     state: started
     enabled: true
+
+- name: "Install logcli"
+  ansible.builtin.unarchive:
+    src: "https://github.com/grafana/loki/releases/download/v\
+             {{ loki_version }}/logcli-linux-amd64.zip"
+    dest: /usr/local/bin
+    remote_src: true
+
+- name: "Rename and set permissions for the Loki binary"
+  ansible.builtin.copy:
+    src: "/usr/local/bin/logcli-linux-amd64"
+    dest: "/usr/local/bin/logcli"
+    owner: root
+    group: root
+    mode: "0755"
+    remote_src: true
```

Comment:
Since it depends on the same version-defining variable as Loki itself, let's move it under the `rescue` block for consistency.